client/src/app/queries/tasks.ts

```diff
@@ -27,7 +28,16 @@ export const useFetchTasks = (
     select: (allTasks) => {
       const uniqSorted = allTasks
         .filter((task) =>
-          filters?.addon ? filters.addon === task.addon : true
+          // If there are any tasks with the addon field, we will still need to consider those older
```

Comment:
I would move it to a separate PR. First, it seems not directly lined with the enhancement. Second, this is really should be re-written with server side filtering (https://github.com/konveyor/tackle2-ui/pull/1956)

Reply:
Moved to #1959